cli/config/compose/profiles/ingest-manager/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     volumes:
       - ${kibanaConfigPath}:/usr/share/kibana/config/kibana.yml
   package-registry:
-    image: docker.elastic.co/package-registry/package-registry:41c150c8020efc53ab16e3bba774c62a419b51ea
```

Comment:
I'm tempted to say we should use here `distribution:snapshot` and use `distribution:staging` for the 7.x e2e tests.

Reply:
I we have this tag, I'm OK with it.

For the 7.x branch, we do not have that branch, only 7.9.x and eventually 7.10.x. Should we have a 7.x branch?

Reply:
As we have 7.* and 8.* branches I assume we are aligned with the release branches. So I think yes. Then it becomes self explaining on what monitors which part.

Reply:
Good. We'd need then to define which versions for the stack are needed, as there is no https://container-library.elastic.co/r/elasticsearch/elasticsearch:7.x image

Reply:
We could use this issue for reference: https://github.com/elastic/e2e-testing/issues/274

Reply:
There actually is one: `docker.elastic.co/elasticsearch/elasticsearch:7.x-SNAPSHOT`